src/server/wesnothd/server.cpp

```diff
@@ -1650,6 +1844,44 @@ void server::handle_player_in_game(player_iterator p, simple_wml::document& data
 
 		send_to_lobby(diff);
 
+		// if this is the creation of a game from a server-side queue, need to tell all the other players in the queue to join
+		if(g.q_type() == queue_type::type::server_preset) {
+			int queue_id = g.queue_id();
+			int game_id = g.id();
+
+			if(queue_info_.count(queue_id) > 0) {
+				queue_info& info = queue_info_.at(queue_id);
+				for(const std::string& name : info.players_in_queue) {
```

Comment:
Shouldn't this loop abort if enough players were told to join?

Reply:
Yeah, true. Updated.